src/Features/VisualBasic/Portable/Completion/VisualBasicCompletionService.vb

```diff
@@ -57,15 +57,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion
             End Get
         End Property
 
-        Private Shared s_defaultCompletionRules As CompletionRules =
-            CompletionRules.Create(
-                dismissIfEmpty:=True,
-                dismissIfLastCharacterDeleted:=True,
-                defaultCommitCharacters:=CompletionRules.Default.DefaultCommitCharacters,
-                defaultEnterKeyRule:=EnterKeyRule.Always)
-
         Public Overrides Function GetRules() As CompletionRules
```

Comment:
Can you follow the same pattern we see in CSharpCompletionService?  i.e. with the interlocking and whatnot.  That will help avoid allocations here.